From a69364e01c76a445fb1078cd504289f0a45b0401 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Wed, 27 Jul 2016 10:09:09 +0800
Subject: [PATCH] clk: rockchip: rk3399: fix up the make warning

fix up the warning:
Line 246: 			rate_table->nf = nf_out;
drivers/clk/rockchip/clk-pll.c:246:19: warning: 'nf_out' may be used uninitialized in this function [-Wmaybe-uninitialized]

Change-Id: I6c11bf91a280f324c21214cb7839e36f9ffa0da3
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-pll.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-pll.c b/drivers/clk/rockchip/clk-pll.c
index 05b40eacaf2f..5f1edeedd17f 100644
--- a/drivers/clk/rockchip/clk-pll.c
+++ b/drivers/clk/rockchip/clk-pll.c
@@ -191,6 +191,7 @@ rockchip_rk3066_pll_clk_set_by_auto(struct rockchip_clk_pll *pll,
 
 	nr_out = PLL_NR_MAX + 1;
 	no_out = 0;
+	nf_out = 0;
 
 	if (fin_hz == 0 || fout_hz == 0 || fout_hz == fin_hz)
 		return NULL;
-- 
2.35.3

